tests/test_management_commands.py

```diff
@@ -0,0 +1,83 @@
+from unittest.mock import patch
+
+from dateutil.relativedelta import relativedelta
+from django.core import mail
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils import timezone
+
+from .factories import UserFactory
+
+
+MONTH = 1
+
+
+class TestUserNotifyManagementCommand(TestCase):
+    def test_no_users(self):
+        call_command('notify_users')
+        self.assertFalse(len(mail.outbox))
+
+    def test_inactive_users(self):
+        month_ago = timezone.now() - relativedelta(months=13)
```

Comment:
I'm not sure `month_ago` is the best name, but I'm not sure what to propose instead.